server/routes/contacts/manage/start/startManageContactsJourneyController.test.ts

```diff
@@ -0,0 +1,139 @@
+import type { Express } from 'express'
+import request from 'supertest'
+import { SessionData } from 'express-session'
+import { v4 as uuidv4 } from 'uuid'
+import { appWithAllRoutes, user } from '../../../testutils/appSetup'
+import AuditService, { Page } from '../../../../services/auditService'
+import ManageContactsJourney = journeys.ManageContactsJourney
+
+jest.mock('../../../../services/auditService')
+
+const auditService = new AuditService(null) as jest.Mocked<AuditService>
+
+let app: Express
+let session: Partial<SessionData>
+let preExistingJourneysToAddToSession: Array<ManageContactsJourney>
+
+beforeEach(() => {
+  app = appWithAllRoutes({
+    services: { auditService },
+    userSupplier: () => user,
+    sessionReceiver: (receivedSession: Partial<SessionData>) => {
+      session = receivedSession
+      if (preExistingJourneysToAddToSession) {
+        session.manageContactsJourneys = {}
+        preExistingJourneysToAddToSession.forEach((journey: ManageContactsJourney) => {
+          session.manageContactsJourneys[journey.id] = journey
+        })
+      }
+    },
+  })
+})
+
+afterEach(() => {
+  jest.resetAllMocks()
```

Comment:
Better to reset `preExistingJourneysToAddToSession` here?
afterEach(() => {
  jest.resetAllMocks();
  preExistingJourneysToAddToSession = []; 
});

Reply:
Done now in next PR, thanks.